.github/CODEOWNERS

```diff
@@ -38,6 +38,7 @@
 /java/ @lidavidm
 /js/ @domoritz @trxcllnt
 /matlab/ @kevingurney @kou @sgilmore10
+/python/ @raulcd
```

Comment:
@rok @AlenkaF let me know if you want to be added here too, as discussed on the pyarrow dev call

Reply:
Sure!
```suggestion
/python/ @raulcd @rok
```